content/docs/rfcs/template/README.md

```diff
@@ -0,0 +1,62 @@
+---
+slug: XX
+title: XX/(WAKU2|LOGOS|CODEX|...)-TEMPLATE
+name: (Waku v2 | Logos | Codex) RFC Template
+status: (raw|draft|stable)
+category: (Standards Track|Informational|Best Current Practice)
+tags: an optional list of tags, not standard
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# Abstract
+
+
+# Background / Rationale / Motivation
+
+This section serves as an introduction providing background information and a motivation/rationale for why the specified protocol is useful.
+
+# Theory / Semantics
+
+A standard track RFC must feature this section.
+This section SHOULD explain in detail how the proposed protocol works.
+It may touch on the wire format where necessary for the explanation.
+This section MAY also specify endpoint behaviour when receiving specific messages, e.g. the behaviour of certain caches etc.
+
+# Wire Format Specification / Syntax
+
+A standard track RFC must feature this section.
```

Comment:
Perhaps only when in `draft` or higher? Perhaps we may want to relax requirements for `raw` RFCs so we can standardise a bit quicker on semantics even if wire format is not yet fleshed out?

Reply:
Yes. The the MUSTs are for "stable". Raw and draft "should". Ill add that.